src/packages/tests/src/__tests__/integration/mssql/__database.ts

```diff
@@ -44,8 +45,14 @@ function getConnectionInfo(ctx: Context) {
     user: 'SA',
     password: 'Pr1sm4_Pr1sm4',
     server: connectionUrl.hostname,
-    port: connectionUrl.port,
+    port: Number(connectionUrl.port),
     database: `master`,
+    pool: {
+      max: 1,
+    },
+    options: {
```

Comment:
I thought this was going to silence this warning `https://github.com/prisma/prisma/runs/1314459154?check_suite_focus=true#step:7:578`, but that doesn't seem to be the case, can you pick up my slack and look at how exactly to silence it? :D

Reply:
Ah nevermind, this is only in Client tests, not related to these changes